client/DotNet/ArmadaProject.Io.Client.Test/Tests.cs

```diff
@@ -0,0 +1,113 @@
+using ApiEvent = ArmadaProject.Io.Api.Event;
+using ArmadaProject.Io.Api;
+using Grpc.Core;
+using Grpc.Net.Client;
+using K8S.Io.Api.Core.V1;
+using K8S.Io.Apimachinery.Pkg.Api.Resource;
+using NUnit.Framework;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArmadaProject.Io.Client.Test
+{
+    public class Tests
+    {
+        [Test]
+        public async Task TestSimpleJobSubmitFlow()
+        {
+            var queue = "test";
+            var jobSetId = Guid.NewGuid().ToString();
+
+            var channel = CreateChannel(new Uri("http://localhost:8080"), ChannelCredentials.Insecure);
+
+            try
+            {
+                //submit
+                var submitClient = new Submit.SubmitClient(channel);
+                var submitRequest = CreateSubmitRequest(queue, jobSetId);
+                var submitResponse = await submitClient.SubmitJobsAsync(submitRequest);
+
+                //watch events
+                var eventClient = new ApiEvent.EventClient(channel);
+                var watchRequest = new WatchRequest
+                {
+                    Queue = queue,
+                    JobSetId = jobSetId
+                };
+                var watchResponse = eventClient.Watch(watchRequest);
+                var submittedEventReceived = false;
+                using (var cts = new CancellationTokenSource())
+                {
+                    cts.CancelAfter(TimeSpan.FromSeconds(30));
+
+                    while (await watchResponse.ResponseStream.MoveNext(cts.Token))
+                    {
+                        if (watchResponse.ResponseStream.Current.Message.EventsCase == EventMessage.EventsOneofCase.Submitted)
+                        {
+                            submittedEventReceived = true;
+                            break;
+                        }
+                    }
+                }
+
+                Assert.That(submittedEventReceived, Is.Not.False);
+            }
+            finally
+            {
+                await channel.ShutdownAsync();
+            }
+        }
+
+        private static GrpcChannel CreateChannel(Uri address, ChannelCredentials credentials)
+        {
+            var options = new GrpcChannelOptions
+            {
+                Credentials = credentials,
+            };
+
+            return GrpcChannel.ForAddress(address, options);
+        }
+
+        private static JobSubmitRequest CreateSubmitRequest(string queue, string jobSetId)
+        {
+            var flex = new FlexVolumeSource();
+            flex.Driver = "gr/cifs";
```

Comment:
Can we avoid putting these 
GR specific things in our public repo?  I'd vote for just simple pod specs with no volumes for our repo.

Reply:
Yes this makes sense. I've removed the volume from the old and new clients.